queries/html_tags/highlights.scm

```diff
@@ -59,3 +59,16 @@
 ] @tag.delimiter
 
 "=" @operator
+
+(attribute
+  ((attribute_name) @_name
+   (#lua-match? @_name "%[.*%]")) @keyword)
+
+(attribute
+  ((attribute_name) @_name
+   (#lua-match? @_name "%(.*%)")) @keyword)
+
+(attribute
+  ((attribute_name) @_name
+   (#lua-match? @_name "^%*.*")) @keyword)
+
```

Comment:
Can this be put inside `angular/highlights.scm`?

Reply:
Actually these aren't needed anymore :)